backend/controllers/vehicleController.js

```diff
@@ -64,15 +75,25 @@ exports.updateVehicle = asyncHandler(async (req, res, next) => {
 exports.createVehicle = asyncHandler(async (req, res, next) => {
   const reqBody = req.body;
 
+  // grab the dealership_ID from the body and verify that the dealership exists
+  const dealership = await Dealership.findById(req.body.dealership);
+
+  // no dealership found
+  if (!dealership) {
+      return next(
+          new ErrorResponse('dealership not found. Cannot create a vehicle without a valid dealership.', 400)
```

Comment:
Would change this message to simply "Cannot create a vehicle without a valid dealership ID"

Reply:
It would not make much difference. However, I updated this line of code to display the invalid dealership_id.